tests/org.pitest.pitclipse.ui.tests/fragment.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <fragment>
-   <extension
```

Comment:
Just for curiosity: here only the order of the extensions has changed... is that important?

Reply:
No, the actual point changed from:
org.pitest.pitclipse.**ui**.results
to:
org.pitest.pitclipse.**core**.results

This is one of the most important changes, because it changes the time and source where this class gets notified.